org.eclipse.jdt.ui.tests/ui/org/eclipse/jdt/ui/tests/jarexport/FatJarExportTests.java

```diff
@@ -124,6 +126,12 @@ public static void setUpTest() {
 	public void setUp() throws Exception {
 		fProject= pts.getProject();
 
+		Map<String, String> options= fProject.getOptions(false);
```

Comment:
Is this on purpose here?

Reply:
It was in despair because of the build issues. Can be removed.

Reply:
Just wanted to delete the file from the patch. In gerrit it was possible. Hmm, should stay away from this while accessing it from a mobile phone. Sorry

Reply:
Fixed